contrib/systemd/labgrid-exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Labgrid Exporter
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/labgrid-exporter /etc/labgrid/exporter.yaml
+Restart=on-abort
+User=labgrid
+Group=labgrid
+SupplementaryGroups=dialout
```

Comment:
The group for which serial ports are available depends on the distribution.

Reply:
Correct, I added a comment that this needs to be adjusted to the distribution. Is it ok?